19-02-2019, SM

# WarmteVerlies Glas
In dit document worden een aantal mogelijke raamverbeteringen met elkaar vergeleken voor wat betreft het warmteverlies, de kosten en de terugverdientijd die hiermee samenhangen. 

## Uitgangspunten

In [1]:
Opp = 2.1*1 + 1.3*2.5 + 2.4*1.3 + 1.8*2.1
print ( "Totaal Oppervlakte = ", Opp )
Lengte   = Opp / 2
Hoogte   = 2

# Veronderstelde temperaturen
Temperatuur_Binnen = 20.0
Temperatuur_Buiten = 0.0

# Voor de berekening van de kostenbesparing gaan we uit van het aantal graaddagen van het vorige jaar.
# De gasprijs in [Euro] per m3:
Price_Gas = 0.80

Totaal Oppervlakte =  12.25


## Materiaal Constanten
Hieronder de definitie van de gebruikte materialen.
<br>Iedere vorm om de warmtegeleiding op te geven is toegestaan ( R-waarde, U-waarde of Lambda + Dikte). 

In [2]:
from Base_Heat import *
import Base_Heat

# luchtweerstand van de binnenmuur, zoals gedefinieerd in het bouwbesluit
Rsi = Heat_Transfer_Object ( "Rsi" )
Rsi.R_Waarde = 0.13

# luchtweerstand van de buitenmuur, zoals gedefinieerd in het bouwbesluit
Rse = Heat_Transfer_Object ( "Rse" )
Rse.R_Waarde = 0.04

HR_Glas = Heat_Transfer_Object ( "HR glas" ) 
HR_Glas.U_Waarde = 2.8

HRxx_Glas = Heat_Transfer_Object ( "HR++ glas" ) 
HRxx_Glas.U_Waarde = 1.1

HRxxx_Glas = Heat_Transfer_Object ( "HR+++ glas" ) 
HRxxx_Glas.U_Waarde = 0.5

#print ( HR_Glas, HRxx_Glas, HRxxx_Glas )

## HR Glas
Dit is het klassiek Thermopane glas van 30 jaar geleden.

In [3]:
HR_Glas.Lengte   = Opp / 2
HR_Glas.Hoogte   = 2

HR_Glas.Temperature_Inside   = Temperatuur_Binnen
HR_Glas.Temperature_Outside  = Temperatuur_Buiten

print ( HR_Glas )

Name        = HR glas
Lengte      = 6.125 [m]
Hoogte      = 2 [m]
Oppervlakte = 12.2 [m2]
R-Waarde    = 0.36 [m2.K/W]
U-Waarde    = 2.80 [W/m2.K]
T-Inside    = 20.0 [Celsius]
T-Outside   = 0.0 [Celsius]
Heat Loss   = 686 [W]
Heat Loss 1m2   = 56 [W]
Energy Cost 1m2 = 10.75 [Euro]
Gasprijs        = 0.80 [Euro]



## HR++ Glas

In [4]:
HRxx_Glas.Lengte   = Opp / 2
HRxx_Glas.Hoogte   = 2

HRxx_Glas.Temperature_Inside   = Temperatuur_Binnen
HRxx_Glas.Temperature_Outside  = Temperatuur_Buiten

print ( HRxx_Glas )

Name        = HR++ glas
Lengte      = 6.125 [m]
Hoogte      = 2 [m]
Oppervlakte = 12.2 [m2]
R-Waarde    = 0.91 [m2.K/W]
U-Waarde    = 1.10 [W/m2.K]
T-Inside    = 20.0 [Celsius]
T-Outside   = 0.0 [Celsius]
Heat Loss   = 269 [W]
Heat Loss 1m2   = 22 [W]
Energy Cost 1m2 = 4.22 [Euro]
Gasprijs        = 0.80 [Euro]



## HR+++ Glas

In [5]:
HRxxx_Glas.Lengte   = Opp / 2
HRxxx_Glas.Hoogte   = 2

HRxxx_Glas.Temperature_Inside   = Temperatuur_Binnen
HRxxx_Glas.Temperature_Outside  = Temperatuur_Buiten

print ( HRxxx_Glas )

Name        = HR+++ glas
Lengte      = 6.125 [m]
Hoogte      = 2 [m]
Oppervlakte = 12.2 [m2]
R-Waarde    = 2.00 [m2.K/W]
U-Waarde    = 0.50 [W/m2.K]
T-Inside    = 20.0 [Celsius]
T-Outside   = 0.0 [Celsius]
Heat Loss   = 122 [W]
Heat Loss 1m2   = 10 [W]
Energy Cost 1m2 = 1.92 [Euro]
Gasprijs        = 0.80 [Euro]



## Kozijnen
Van een typisch kozijn (3 ramen aan de voorkant, waarvan 1 draaibaar) is het oppervlakte geschat t.o.v. de glas oppervlakte. Dit percentage is gebruikt voor de gehele beglazing. Het draaibare deel komt slechts 2 keer voor en wortd hier verwaarloosd. De tuindeuren tellen niet mee omdat die worden verdikt en apart worden geteld.
<br><br> Vaak (en zeker door leveranciers van kunststof kozijnen) wordt voor hout de forfaitaire waarde van U = 2.4 [W/m2K] aangehouden. Waar deze waarde vandaan komt weet ik niet maar ik kan geen houtsoort vinden die zo'n slechte isolatie-waarde heeft.
<br>In deze link staan van een groot aantal gangbare houtsoorten de isolatie weerstand: 
https://ph-bouwadvies.nl/u-waarden-van-kozijnen-en-de-energieprestatie/

Een online berekentool voor kozijnen is hier te vinden: http://www.warmteweerstand.nl/U-waarde_kozijn.aspx
<br>Deze tool geeft een veel realistischers waarde voor de isolatie van houten kozijnen, maar ook hier heb ik twijfels over het gebruikte reken-algoritme. Het verschil tussen Grenen (1.36) en Merbau (1.44) is namelijk veel kleiner dan op grond van de lambda mag worden verwacht.

In [6]:
Lambda_Merbau           = 0.18  # geldt ook voo Afzalia, Bintanger, Robinia, Eiken, Supupira
Lambda_Dark_Red_Meranti = 0.16  # geldt ook voor Iroko, Sapeli, Sipo,Teak
Lambda_Grenen           = 0.13  # geldt ook Led-Red-Meranti, Lariks, Oregon-Pine, Accoya
Lambda_Vuren            = 0.11  # geldt ook voor Western-Red-Cedar

In [7]:
Voorraam_Opp = 2.1 * 0.98
Glas_Opp     = 3 * 0.61 * 0.87
Draai_Opp    = 2 * 2 * 0.05 * ( 0.60 + 0.80 )

Kozijn_Opp1      = Voorraam_Opp - Glas_Opp
Kozijn_Opp1_Perc = 100* Kozijn_Opp1 / Voorraam_Opp

print ( "Voorraam_Opp     = %.2f [m2]" % Voorraam_Opp )
print ( "Kozijn_Opp1      = %.2f [m2]" % Kozijn_Opp1 )
print ( "Draai_Opp        = %.2f [m2]" % Draai_Opp )
print ( "Kozijn_Opp1_Perc = %.0f %%"   % Kozijn_Opp1_Perc )

Voorraam_Opp     = 2.06 [m2]
Kozijn_Opp1      = 0.47 [m2]
Draai_Opp        = 0.28 [m2]
Kozijn_Opp1_Perc = 23 %


In [8]:
Kozijn_Dikte = 0.12
Draai_Dikte  = 0.05

Kozijn_Opp = Kozijn_Opp1_Perc * Opp / 100

Kozijn_Merbau        = Heat_Transfer_Object ( "Kozijn Merbau" )  
Kozijn_Merbau.Lambda = Lambda_Merbau
Kozijn_Merbau.Dikte  = Kozijn_Dikte

Kozijn_Dark_Red_Meranti        = Heat_Transfer_Object ( "Kozijn Dark Red Merantie" )  
Kozijn_Dark_Red_Meranti.Lambda = Lambda_Dark_Red_Meranti
Kozijn_Dark_Red_Meranti.Dikte  = Kozijn_Dikte

Kozijn_Grenen        = Heat_Transfer_Object ( "Kozijn Grenen" )  
Kozijn_Grenen.Lambda = Lambda_Grenen
Kozijn_Grenen.Dikte  = Kozijn_Dikte

Kozijn_Vuren        = Heat_Transfer_Object ( "Kozijn Vuren" )  
Kozijn_Vuren.Lambda = Lambda_Vuren
Kozijn_Vuren.Dikte  = Kozijn_Dikte

Kozijn_HH1 = Sandwich ( "Kozijn Merbau" )
Kozijn_HH1.Lengte              = 2
Kozijn_HH1.Breedte             = Kozijn_Opp / Kozijn_HH1.Lengte
Kozijn_HH1.Temperature_Inside  = Temperatuur_Binnen
Kozijn_HH1.Temperature_Outside = Temperatuur_Buiten
Kozijn_HH1.Add ( Rsi )
Kozijn_HH1.Add ( Kozijn_Merbau )
Kozijn_HH1.Add ( Rse )

Kozijn_HH2 = Sandwich ( "Kozijn Dark Red Meranti" )
Kozijn_HH2.Lengte              = 2
Kozijn_HH2.Breedte             = Kozijn_Opp / Kozijn_HH2.Lengte
Kozijn_HH2.Temperature_Inside  = Temperatuur_Binnen
Kozijn_HH2.Temperature_Outside = Temperatuur_Buiten
Kozijn_HH2.Add ( Rsi )
Kozijn_HH2.Add ( Kozijn_Dark_Red_Meranti )
Kozijn_HH2.Add ( Rse )

Kozijn_HH3 = Sandwich ( "Kozijn Grenen" )
Kozijn_HH3.Lengte              = 2
Kozijn_HH3.Breedte             = Kozijn_Opp / Kozijn_HH3.Lengte
Kozijn_HH3.Temperature_Inside  = Temperatuur_Binnen
Kozijn_HH3.Temperature_Outside = Temperatuur_Buiten
Kozijn_HH3.Add ( Rsi )
Kozijn_HH3.Add ( Kozijn_Grenen )
Kozijn_HH3.Add ( Rse )

Kozijn_HH4 = Sandwich ( "Kozijn Vuren" )
Kozijn_HH4.Lengte              = 2
Kozijn_HH4.Breedte             = Kozijn_Opp / Kozijn_HH4.Lengte
Kozijn_HH4.Temperature_Inside  = Temperatuur_Binnen
Kozijn_HH4.Temperature_Outside = Temperatuur_Buiten
Kozijn_HH4.Add ( Rsi )
Kozijn_HH4.Add ( Kozijn_Vuren )
Kozijn_HH4.Add ( Rse )

Base_Heat.PrintLevel = 1

print ( Kozijn_HH1 )
print ( Kozijn_HH2 )
print ( Kozijn_HH3 )
print ( Kozijn_HH4 )

Name        = Kozijn Merbau
Layers    = [ Rsi, Kozijn Merbau, Rse,  ]
R-Waarden = 0.84 [ 0.13, 0.67, 0.04,  ] [m2.K/W]

Name        = Kozijn Dark Red Meranti
Layers    = [ Rsi, Kozijn Dark Red Merantie, Rse,  ]
R-Waarden = 0.92 [ 0.13, 0.75, 0.04,  ] [m2.K/W]

Name        = Kozijn Grenen
Layers    = [ Rsi, Kozijn Grenen, Rse,  ]
R-Waarden = 1.09 [ 0.13, 0.92, 0.04,  ] [m2.K/W]

Name        = Kozijn Vuren
Layers    = [ Rsi, Kozijn Vuren, Rse,  ]
R-Waarden = 1.26 [ 0.13, 1.09, 0.04,  ] [m2.K/W]



## Vergelijk van de verschillende Glassoorten
Deze tabel functie is gemaakt om het warmteverlies van een gehele ruimte of woning in detail weer te geven. In de huidige situatie waar een aantal verschillende glasssoorten met elkaar worden vergeleken, is de onderste regel "Totaal", evenals enkele kolommen, volkomen zinloos. 
<br>Het rechter deel van de tabel (kolommen met de getallen erboven) geeft de kolomnaam de buitentemperatuur aan en de waarden in die kolom staan, is het warmteverlies in Watt dat door de muur weglekt bij die buitentemperatuur. 
<br>kolom Kosten    = de geschatte totale jaarlijke Energiekosten in Euros (bij gebruik van Gas)
<br>kolom Kosten-m2 = de geschatte jaarlijke Energiekosten per m2 in Euros (bij gebruik van Gas)

In [9]:
Ts  = range ( -8, 12, 2 )
Elements = [ HR_Glas, HRxx_Glas, HRxxx_Glas, Kozijn_HH1, Kozijn_HH2, Kozijn_HH3, Kozijn_HH4 ]
Heat_Transfer_Table ( Elements, Ts )

R-waarde  U-waarde  Oppervlak  T-Binnen  T-Buiten  \
HR glas                      0.35       2.8      12.25        20         0   
HR++ glas                     0.9       1.1      12.25        20         0   
HR+++ glas                      2       0.5      12.25        20         0   
Kozijn Merbau                0.83      1.19       2.77        20         0   
Kozijn Dark Red Meranti      0.92      1.08       2.77        20         0   
Kozijn Grenen                1.09      0.91       2.77        20         0   
Kozijn Vuren                 1.26      0.79       2.77        20         0   
Totaal                   R-waarde  U-waarde  Oppervlak  T-Binnen  T-Buiten   

                         Kosten-m2 Kosten Jaar-m3    -8    -6    -4    -2  \
HR glas                      14.91    182     228   960   891   823   754   
HR++ glas                     5.85     71      89   377   350   323   296   
HR+++ glas                    2.66     32      40   171   159   147   134   
Kozijn Merbau                 6.36     17      22    92    86    79    72   
Kozijn Dark Red Meranti       5.78     16      20    84    78    72    66   
Kozijn Grenen                 4.87     13      16    71    65    60    55   
Kozijn Vuren                  4.22     11      14    61    57    52    48   
Totaal                   Kosten-m2    342     429  1816  1686  1556  1425   

                            0     2     4    6    8   10  
HR glas                   686   617   548  480  411  343  
HR++ glas                 269   242   215  188  161  134  
HR+++ glas                122   110    98   85   73   61  
Kozijn Merbau              66    59    53   46   39   33  
Kozijn Dark Red Meranti    60    54    48   42   36   30  
Kozijn Grenen              50    45    40   35   30   25  
Kozijn Vuren               43    39    35   30   26   21  
Totaal                   1296  1166  1037  906  776  647

## Conclusie

HR++ glas is ongeveer 3 keer beter dan HR glas.
<br>HR++= glas is ongeveer 6 keer beter dan HR glas.


Vanuit: https://www.verbouwkosten.com/dubbel-glas/kosten/#Per-m2
<br>Van thermopane (het oude gewone dubbel glas) naar HR++ glas bespaart 10m3 gas / m2, hetgeen overeenkomt met 8 Euro / m2.
<br>Installatie kost (exclusief afvoer en schilderen) 135 Euro / m2, wat neer komt op een terugverdientijd van 17 jaar !!
<br>De kosten bestaan ruwweg voor de helft uit materiaalkosten en voor de helft uit arbeidskosten. Als we nu inschatten dat het schilderwerk ook nog eens dezelfde arbeidskosten vereist, komen we in totaal op een terugverdientijd van ongeveer 25 jaar.

Op basis van mijn eigen berekeningen kom ik nog iets ongunstiger uit, namelijk een besparing van 6.5 Eur / m2,
oftewel een terugverdientijd van 20 jaar zonder schilderwerk en 30 jaar met schilderwerk!!!

Bij zelfdoen kom je dus uit op ongeveer de helft van de kosten en dus ook op ongeveer de helft van de terugverdientijd.

Dus gooi het over de boeg van comfort !!!!
<br><img src="Glas_Temperaturen.png" width="50%" height="5%">

Daarnaast is het waarschijnlijk interessant om (als het kozijn dat toelaat), meteen over te gaan naar HR+++ glas)

Kozijnen maken toch een substantieel deel uit van het totale buitenoppervlak, in het algemeen 15 .. 20 % en in mijn geval zelfs 23% van het glas oppervlak. Op websites van leveranciers van kozijnen wordt vaak gerekend met een forfaitaire waarde van de isolatie-waarde van hout, waardoor een te slechte voorstelling van zaken wordt gegeven. Zoals de table hierboven laat zien, hebben alle houtsoorten een vergelijkbaarwarmteverlies als HR++ glas. Alleen in het geval dat HR+++ glas wordt genomen dienen andere kozijnen gebruikt te worden. De economische besparing van betere kozijnen bedraagt in mijn geval ongeveer 10 Euro per jaar.